BSseq_pipeline.py

```diff
@@ -489,35 +489,37 @@
         os.path.join(DIR_methcall,"{prefix}.sorted_meth_calls.log")
     message: fmt("Extract methylation calls from bam file.")
     shell:
-        nice(" ".join(["{RSCRIPT}","{DIR_scripts}/methCall.R",
-        "--inBam={params.inBam}",
-        "--assembly={params.assembly}",
-        "--mincov={params.mincov}",
-        "--minqual={params.minqual}",
-        "--rds={params.rds}",
-        "--logFile={log}"]))
+        nice('Rscript', ["{DIR_scripts}/methCall.R",
```

Comment:
much better!

I feel that these attempts could be squashed to one commit that does it right the first time.